doc/classes/Particles2D.xml

```diff
@@ -305,26 +305,27 @@
 			Number of horizontal frames in [code]texture[/code].
 		</member>
 		<member name="lifetime" type="float" setter="set_lifetime" getter="get_lifetime">
-			Amount of time each particle will exist. Default value: [code]1[/code].
+			Amount of seconds each particle will exist. Default value: [code]1[/code].
 		</member>
 		<member name="local_coords" type="bool" setter="set_use_local_coordinates" getter="get_use_local_coordinates">
 			If [code]true[/code] particles use the parent node's coordinate space. If [code]false[/code] they use global coordinates. Default value: [code]true[/code].
 		</member>
 		<member name="normal_map" type="Texture" setter="set_normal_map" getter="get_normal_map">
 		</member>
 		<member name="one_shot" type="bool" setter="set_one_shot" getter="get_one_shot">
-			If [code]true[/code] only [code]amount[/code] particles will be emitted. Default value: [code]false[/code].
+			If [code]true[/code] does only one emission cycle or if set [true] while one is ongoing, doesn't start the next one. Default value: [code]false[/code].
 		</member>
 		<member name="preprocess" type="float" setter="set_pre_process_time" getter="get_pre_process_time">
+			Particle system starts as if it had already run for this many seconds.
 		</member>
 		<member name="process_material" type="Material" setter="set_process_material" getter="get_process_material">
 			[Material] for processing particles. Can be a [ParticlesMaterial] or a [ShaderMaterial].
 		</member>
 		<member name="randomness" type="float" setter="set_randomness_ratio" getter="get_randomness_ratio">
-			Emission randomness ratio. Default value: [code]0[/code].
+			Emission lifetime randomness ratio. Default value: [code]0[/code].
 		</member>
 		<member name="speed_scale" type="float" setter="set_speed_scale" getter="get_speed_scale">
-			Speed scaling ratio. Default value: [code]1[/code].
+			Particle system's speed scaling ratio. [code]0.01[/code] makes particles crawl and [code]2[/code] makes them fly twice as fast. Everything scales, so they still fly as far. Is not particle's [code]initial_velocity[/code], which is found in [ParticlesMaterial] Default value: [code]1[/code].
```

Comment:
This is way too informal. Documentation should be clear and concise, not conversational. It's also inaccurate - "fly" is not a good verb to use because particles can behave in lots of ways - they may fall, float, bounce, or not even move at all. If I were just scaling particles, the speed scale would still apply without movement.

Reply:
Fixed.